reporting/jolt-spec/reference-data-jolt-spec.json

```diff
@@ -65,8 +65,8 @@
       "joltspec": [{"operation":"shift","spec":{"id":"id","description":"description","supervisoryNode":{"id":"supervisorynodeid"},"program":{"id":"programid"},"supplyingFacility":{"id":"supplyingfacilityid"}}}]
     },
     {
-      "endpoint": "stockAdjustmentReasons",
+      "endpoint": "validReasons",
```

Comment:
Can you check why this entity has changes in code? @anawrotsoldevelo Have you changes anything in this section?

Reply:
Change in endpoint name for stockAdjustmentReasons was added by a mistake. I reverted it to previous state